stackgl_modules/package.json

```diff
@@ -13,21 +13,22 @@
     "box-intersect": "plotly/box-intersect#v1.1.0",
     "convex-hull": "^1.0.3",
     "delaunay-triangulate": "^1.1.6",
-    "gl-axes3d": "^1.7.0",
-    "gl-cone3d": "^1.6.0",
-    "gl-error3d": "^1.0.16",
+    "gl-axes3d": "github:hborchardt/gl-axes3d#83caafc058b2d60b6006e26432967c411c125fe1",
+    "gl-cone3d": "github:hborchardt/gl-cone3d#e9728cd068e0dfe11b92c5785974cd463f78c01c",
+    "gl-error3d": "github:hborchardt/gl-error3d#ee55620d17f3cc7dfb104823c904f5070aba79ed",
     "gl-heatmap2d": "^1.1.1",
-    "gl-line3d": "1.2.1",
-    "gl-mesh3d": "^2.3.1",
+    "gl-line3d": "github:hborchardt/gl-line3d#27123949f65e2d8deab6a6e131030073ed9cdf02",
+    "gl-mesh3d": "github:hborchardt/gl-mesh3d#be2c7fcb68e70e001bbfb8021380d0d7ac75b1c9",
     "gl-plot2d": "^1.5.0",
     "gl-plot3d": "^2.4.7",
     "gl-pointcloud2d": "^1.0.3",
-    "gl-scatter3d": "^1.4.0",
+    "gl-scatter3d": "github:hborchardt/gl-scatter3d#086177000f405bc72f171350f9c1efa6eb523e29",
     "gl-select-box": "^1.0.4",
     "gl-shader": "4.3.1",
     "gl-spikes2d": "^1.0.2",
-    "gl-streamtube3d": "^1.4.1",
-    "gl-surface3d": "^1.6.0",
+    "gl-spikes3d": "github:hborchardt/gl-spikes3d#9ad06b2dd48fa19d2faa54e5645c2e4bf5e6157c",
+    "gl-streamtube3d": "github:hborchardt/gl-streamtube3d#29c914cc0d88fc5dffce62dcd8169f38115d078b",
+    "gl-surface3d": "github:hborchardt/gl-surface3d#0728e33c0296c602b147e8f1ea8f1bfe7040c61e",
```

Comment:
See https://github.com/gl-vis/gl-surface3d/pull/20

Reply:
Please install the patch.
```
npm i gl-surface3d@1.6.1
```